# Teleop Basic Param

navigation:
  xy_control_mode: 0
  # World Pos Control Mode: 0
  # World Vel Control Mode: 2
  # Local Vel Control Mode: 3
  # Attitude Control Mode: 4


  takeoff_height: 1.0 #1.0
  outdoor_takeoff_height: 1.5
  convergent_duration: 10.0
  land_descend_vel: -0.15

  # teleop operation
  max_target_vel: 0.5
  joy_target_vel_interval: 0.005  # 0.2 / 20 = 0.01, 0.005 ~ 0.01  m/s
  joy_target_z_interval: 0.01
  max_target_yaw_rate: 0.1 #  0.05
  max_target_roll_pitch_rate: 0.0005 #  0.05
  teleop_local_frame: fc
  force_landing_to_halt_du: 0.01

  cmd_vel_lev2_gain : 2.0
  nav_vel_limit : 0.2
  vel_nav_threshold : 0.05

  gain_tunning_mode: 0
  max_target_tilt_angle: 0.2
  cmd_angle_lev2_gain : 1.5

  # gps waypoint
  gps_waypoint_threshold: 3.0
  gps_waypoint_check_du: 1.0

  # desire tilt
  baselink_rot_pub_interval: 0.5
  baselink_rot_change_thresh: 0.01 # 0.2 hz -> 0.04; 0.01 hz-> 0.02
